go/backend/archive/archive.go

```diff
@@ -60,6 +65,9 @@ type Archive interface {
 	// CreateWitnessProof creates a witness proof for the given account and keys.
 	CreateWitnessProof(block uint64, address common.Address, keys ...common.Key) (witness.Proof, error)
 
+	// CreateLiveDBGenesis writes a LiveDB genesis for given block into out.
+	CreateLiveDBGenesis(block uint64, out io.Writer) (common.Hash, error)
```

Comment:
As in the Carmen interface, I would argue that what is exported is not a `Genesis`.